src/test/suite/commands/runDraftTool/helper/runDraftHelper.test.ts

```diff
@@ -0,0 +1,28 @@
+import * as assert from 'assert';
+import {
+   ensureDraftBinary,
+   getDraftConfig,
+   runDraftCommand
+} from '../../../../../commands/runDraftTool/helper/runDraftHelper';
+import {
+   getAsyncResult,
+   failed,
+   succeeded
+} from '../../../../../utils/errorable';
+
+suite('Run Draft Helper Test Suite', () => {
+   test('it returns true when draft binary exists', async () => {
+      var ensureDraftBinaryResult = await ensureDraftBinary();
```

Comment:
You should pretty much never use `var` with modern JavaScript. https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Always prefer `const` > `let` > `var`

Reply:
thanks for pointing out! I've changed them to const.